Switch Composer docs from WPackagist to WP Packages (#10063)
pantheon-systems/documentation: src/source/content/guides/integrated-composer/03-dependencies.md

[Packagist](https://packagist.org) is a repository of Composer packages that are available by default to projects managed by Composer. Packagist libraries receive updates from their source GitHub repositories automatically.

[WP Packages](https://wp-packages.org) is a Packagist-like mirror of the WordPress.org [plugin](https://wordpress.org/plugins) and [theme](https://wordpress.org/themes) repositories and is included with Bedrock out of the box.

> [WPackagist](https://wpackagist.org) is another longstanding Composer mirror for WordPress plugins and themes maintained by [WPEngine in 2026](https://wpengine.com/blog/wp-engine-acquires-wpackagist/).

Reviewer: ```suggestion
[WPackagist](https://wpackagist.org) is another longstanding Composer mirror for WordPress plugins and themes acquired by [WPEngine in 2026](https://wpengine.com/blog/wp-engine-acquires-wpackagist/).
```

Reviewer 2: @stevector I felt like documenting acquisitions was putting our thumb on the needle a little more than I'd like, but I recognize that "maintained by __ in ___" sounds off. what about:

```suggestion
[WPackagist](https://wpackagist.org) is another longstanding Composer mirror for WordPress plugins and themes maintained by [WPEngine since April 2026](https://wpengine.com/blog/wp-engine-acquires-wpackagist/).
```

?

[Packagist](https://packagist.org) is a repository of Composer packages that are available by default to projects managed by Composer. Packagist libraries receive updates from their source GitHub repositories automatically.

> [WP Packages](https://wp-packages.org) is a Packagist-like mirror of the WordPress.org [plugin](https://wordpress.org/plugins) and [theme](https://wordpress.org/themes) repositories and is included with Bedrock out of the box.

Reviewer 2: ```suggestion
[WP Packages](https://wp-packages.org) is a Packagist-like mirror of the WordPress.org [plugin](https://wordpress.org/plugins) and [theme](https://wordpress.org/themes) repositories and is included with Bedrock out of the box.

[WPackagist](https://wpackagist.org) is another longstanding Composer mirror for WordPress plugins and themes maintained by [WPEngine in 2026](https://wpengine.com/blog/wp-engine-acquires-wpackagist/).
```

> You can install packages from Packagist or WP Packages without any additional configuration using `composer require`.

Reviewer 2: ```suggestion
You can install packages from Packagist, WPackagist or WP Packages without any additional configuration using `composer require`.
```

Packages that are flagged as `wordpress-plugin`, `wordpress-theme` or `wordpress-muplugin` in their `composer.json` files will be installed automatically in the appropriate `web/app/` directory by Composer.

> ##### Requiring a package from WP Packages

Reviewer 2: ```suggestion
##### Requiring a package from WP Packages or WPackagist
```

> For all other plugins and themes that are not managed on Packagist, you can use `composer require` as well, using `wp-plugin` or `wp-theme` as the vendor and the plugin or theme slug as the package name.

Reviewer 2: ```suggestion
For all other plugins and themes that are not managed on Packagist, you can use `composer require` as well, using `wp-plugin`/`wp-theme` (for WP Packages) or `wpackagist-plugin`/`wpackagist-theme` (for WPackagist) as the vendor and the plugin or theme slug as the package name.

<Alert title="Choosing a Composer repository or using multiple" type="info">

It is possible to use multiple different Composer repositories, e.g. WPackagist and WP Packages side-by-side. Since they use different vendor names for plugins and themes (`wpackagist-*` and `wp-*` respectively), there is no conflict in using both simultaneously. However, there's no specific benefit in using both. It's recommended to choose one provider in your `composer.json` file and using that provider's vendor prefix for WordPress plugins or themes.

</Alert>
```

```bash{promptUser: user}
composer require wp-theme/twentytwentytwo
```

```bash{promptUser: user}
composer require wp-plugin/advanced-custom-fields
```

##### Check first

> It's generally a good idea when using either Packagist or WP Packages to check the repository before `require`ing the package. If you search Packagist for a WordPress plugin or theme and don't see it, you can be sure that if it exists in the WordPress plugin or theme repository, it will be available on WP Packages. Checking WP Packages for the package can be beneficial if you want to check what versions are available.

Reviewer 2: ```suggestion
It's generally a good idea when using either Packagist, WP Packages or WPackagist to check the repository before `require`ing the package. If you search Packagist for a WordPress plugin or theme and don't see it, you can be sure that if it exists in the WordPress plugin or theme repository, it will be available on WP Packages or WPackagist. Checking the WordPress package repositories for the package can be beneficial if you want to check what versions are available.
```
